packages/semantic-ui/src/CheckboxWidget/CheckboxWidget.tsx

```diff
@@ -52,7 +48,7 @@ function CheckboxWidget(props: WidgetProps) {
       onBlur={_onBlur}
       onFocus={_onFocus}
       required={required}
-      label={displayLabel ? label || schema.title : false}
+      label={label || ""}
```

Comment:
Why not keep `|| schema.title`?

Reply:
want to keep it consistent like the other themes

Reply:
Ah, yeah, not every theme is the same (bootstrap 4 does something different), but this works.